modules/about-the-single-model-serving-platform.adoc

```diff
@@ -30,6 +30,7 @@ When you have installed KServe, you can use the {productname-short} dashboard to
 
 * *TGIS Standalone ServingRuntime for KServe*: A runtime for serving TGI-enabled models
 * *Caikit-TGIS ServingRuntime for KServe*: A composite runtime for serving models in the Caikit format
+* *Caikit Standalone ServingRuntime for KServe*: A runtime for representing text in a numerical format in a vector database that can be used in Retrieval Augmented Generation (RAG) use cases.
```

Comment:
I would replace this with:

"A runtime for serving models in the Caikit Embeddings format for embeddings tasks."

While it can certainly be used for RAG, there are a variety other contexts that it can be used in.